spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -629,6 +629,38 @@
         end
       end
     end
+
+    def stub_auth
+      allow(controller).to receive(:validate_saml_request).and_return(true)
```

Comment:
For potential reuse when I work on speeding up tests so that they don't use the DB if they don't have to.